2020.11.02 14:56:10 INFO  started: Metals version 0.9.4 in workspace '/home/seth/Desktop/Scala_Labs' for client vscode.
2020.11.02 14:56:13 INFO  time: initialize in 3.07s
2020.11.02 14:56:15 WARN  no build tool detected in workspace '/home/seth/Desktop/Scala_Labs'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
2020.11.02 15:19:41 INFO  no build target: using presentation compiler with only scala-library
2020.11.02 15:19:41 WARN  no build target for: /home/seth/Desktop/Scala_Labs/HelloWorld.sc
2020.11.02 15:19:41 WARN  no build target for: /home/seth/Desktop/Scala_Labs/HelloWorld.sc
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 2
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 6
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 8
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 7
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 9
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 11
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 13
Nov 02, 2020 3:19:41 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 15
something's wrong: no file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc, 37, 37, 52)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc, 37, 37, 48)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc, 37, 37, 49)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc in Array[String]RangePosition(file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc, 37, 37, 50)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc in Array[String]RangePosition(file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc, 37, 37, 50)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc in Array[String]RangePosition(file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc, 37, 37, 50)
2020.11.02 15:20:38 WARN  no build target for: /home/seth/Desktop/Scala_Labs/HelloWorld.sc
2020.11.02 15:22:21 INFO  Ammonite: Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true

2020.11.02 15:22:21 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /home/seth/.cache/metals/bsp.trace.json
2020.11.02 15:22:23 INFO  Ammonite: Starting BSP server
2020.11.02 15:22:23 INFO  Ammonite: 

2020.11.02 15:22:23 INFO  Connected to Ammonite Build server v2.2.0-4-4bd225e
2020.11.02 15:22:39 WARN  Could not find java sources in None. Java symbols will not be available.
2020.11.02 15:22:42 INFO  time: indexed workspace in 3.94s
2020.11.02 15:23:14 INFO  compiling helloworld.sc
2020.11.02 15:23:18 INFO  time: compiled HelloWorld.sc in 3.92s
Nov 02, 2020 3:37:18 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleNotification
WARNING: Notification threw an exception: {
  "jsonrpc": "2.0",
  "method": "workspace/didChangeWatchedFiles",
  "params": {
    "changes": [
      {
        "uri": "file:///home/seth/Desktop/Scala_Labs/HelloWorld.sc",
        "type": 3
      }
    ]
  }
}
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.nio.file.NoSuchFileException: /home/seth/Desktop/Scala_Labs/HelloWorld.sc
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:370)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:421)
	at java.base/java.nio.file.Files.readAllBytes(Files.java:3205)
	at scala.meta.internal.io.PlatformFileIO$.slurp(PlatformFileIO.scala:45)
	at scala.meta.internal.io.FileIO$.slurp(FileIO.scala:24)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$onChange$1(MetalsLanguageServer.scala:1077)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$onChange$1$adapted(MetalsLanguageServer.scala:1076)
	at scala.collection.immutable.Stream.foreach(Stream.scala:533)
	at scala.meta.internal.metals.MetalsLanguageServer.onChange(MetalsLanguageServer.scala:1076)
	at scala.meta.internal.metals.MetalsLanguageServer.didChangeWatchedFiles(MetalsLanguageServer.scala:1028)
	... 16 more

2020.11.02 15:38:28 INFO  disconnected: ammonite build server
2020.11.02 15:38:30 ERROR timeout: build server 'Ammonite' during shutdown
2020.11.02 15:38:31 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Celcius.sc
2020.11.02 15:38:31 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /home/seth/.cache/metals/bsp.trace.json
2020.11.02 15:38:31 INFO  Ammonite: Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true

2020.11.02 15:38:34 INFO  Ammonite: Starting BSP server
2020.11.02 15:38:34 INFO  Ammonite: 

2020.11.02 15:38:34 INFO  Connected to Ammonite Build server v2.2.0-4-4bd225e
2020.11.02 15:38:50 WARN  Could not find java sources in None. Java symbols will not be available.
2020.11.02 15:38:50 INFO  time: indexed workspace in 2.05s
2020.11.02 15:38:51 INFO  compiling lab2/celcius.sc
2020.11.02 15:38:55 INFO  time: compiled Celcius.sc in 4.33s
Nov 02, 2020 3:39:09 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 243
Nov 02, 2020 3:39:09 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 244
Nov 02, 2020 3:39:09 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 242
Nov 02, 2020 3:39:09 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 245
Nov 02, 2020 3:39:09 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 246
2020.11.02 15:46:05 INFO  compiling lab2/celcius.sc
2020.11.02 15:46:05 INFO  time: compiled Celcius.sc in 0.5s
Nov 02, 2020 3:47:00 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleNotification
WARNING: Notification threw an exception: {
  "jsonrpc": "2.0",
  "method": "textDocument/didChange",
  "params": {
    "textDocument": {
      "version": 241,
      "uri": "file:///home/seth/Desktop/Scala_Labs/Lab2/Celcius.sc"
    },
    "contentChanges": [
      {
        "text": "\n\n"
      }
    ]
  }
}
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: -1 is not a valid offset, allowed [0..2]
	at scala.meta.internal.inputs.InternalInput.offsetToLine(InternalInput.scala:49)
	at scala.meta.internal.inputs.InternalInput.offsetToLine$(InternalInput.scala:43)
	at scala.meta.inputs.Input$VirtualFile.offsetToLine(Input.scala:80)
	at scala.meta.inputs.Position$Range.startLine(Position.scala:37)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionMetaPosition.toLSP(CommonMtagsEnrichments.scala:74)
	at scala.meta.internal.metals.TokenEditDistance.toRevised(TokenEditDistance.scala:135)
	at scala.meta.internal.metals.Diagnostics.toFreshDiagnostic(Diagnostics.scala:229)
	at scala.meta.internal.metals.Diagnostics.$anonfun$publishDiagnostics$3(Diagnostics.scala:197)
	at scala.meta.internal.metals.Diagnostics.$anonfun$publishDiagnostics$3$adapted(Diagnostics.scala:196)
	at scala.collection.Iterator.foreach(Iterator.scala:943)
	at scala.collection.Iterator.foreach$(Iterator.scala:943)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at scala.meta.internal.metals.Diagnostics.publishDiagnostics(Diagnostics.scala:196)
	at scala.meta.internal.metals.Diagnostics.publishDiagnostics(Diagnostics.scala:174)
	at scala.meta.internal.metals.Diagnostics.didChange(Diagnostics.scala:122)
	at scala.meta.internal.metals.MetalsLanguageServer.didChange(MetalsLanguageServer.scala:900)
	... 15 more

Nov 02, 2020 3:47:14 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
Nov 02, 2020 3:47:14 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: null
java.lang.NullPointerException
	at scala.reflect.internal.Definitions$DefinitionsClass.isByNameParamType(Definitions.scala:417)
	at scala.reflect.internal.TreeInfo.isStableIdent(TreeInfo.scala:138)
	at scala.reflect.internal.TreeInfo.isStableIdentifier(TreeInfo.scala:113)
	at scala.reflect.internal.TreeInfo.isPath(TreeInfo.scala:102)
	at scala.reflect.internal.TreeInfo.admitsTypeSelection(TreeInfo.scala:156)
	at scala.tools.nsc.interactive.Global.stabilizedType(Global.scala:959)
	at scala.tools.nsc.interactive.Global.typedTreeAt(Global.scala:807)
	at scala.meta.internal.pc.AutoImportsProvider.autoImports(AutoImportsProvider.scala:26)

Nov 02, 2020 3:47:19 PM scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
INFO: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
Nov 02, 2020 3:47:19 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: null
java.lang.NullPointerException
	at scala.reflect.internal.Definitions$DefinitionsClass.isByNameParamType(Definitions.scala:417)
	at scala.reflect.internal.TreeInfo.isStableIdent(TreeInfo.scala:138)
	at scala.reflect.internal.TreeInfo.isStableIdentifier(TreeInfo.scala:113)
	at scala.reflect.internal.TreeInfo.isPath(TreeInfo.scala:102)
	at scala.reflect.internal.TreeInfo.admitsTypeSelection(TreeInfo.scala:156)
	at scala.tools.nsc.interactive.Global.stabilizedType(Global.scala:959)
	at scala.tools.nsc.interactive.Global.typedTreeAt(Global.scala:807)
	at scala.meta.internal.pc.AutoImportsProvider.autoImports(AutoImportsProvider.scala:26)

Nov 02, 2020 3:48:01 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleNotification
WARNING: Notification threw an exception: {
  "jsonrpc": "2.0",
  "method": "textDocument/didChange",
  "params": {
    "textDocument": {
      "version": 354,
      "uri": "file:///home/seth/Desktop/Scala_Labs/Lab2/Celcius.sc"
    },
    "contentChanges": [
      {
        "text": "\n\n"
      }
    ]
  }
}
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 11 more
Caused by: java.lang.IllegalArgumentException: -1 is not a valid offset, allowed [0..2]
	at scala.meta.internal.inputs.InternalInput.offsetToLine(InternalInput.scala:49)
	at scala.meta.internal.inputs.InternalInput.offsetToLine$(InternalInput.scala:43)
	at scala.meta.inputs.Input$VirtualFile.offsetToLine(Input.scala:80)
	at scala.meta.inputs.Position$Range.startLine(Position.scala:37)
	at scala.meta.internal.mtags.CommonMtagsEnrichments$XtensionMetaPosition.toLSP(CommonMtagsEnrichments.scala:74)
	at scala.meta.internal.metals.TokenEditDistance.toRevised(TokenEditDistance.scala:135)
	at scala.meta.internal.metals.Diagnostics.toFreshDiagnostic(Diagnostics.scala:229)
	at scala.meta.internal.metals.Diagnostics.$anonfun$publishDiagnostics$3(Diagnostics.scala:197)
	at scala.meta.internal.metals.Diagnostics.$anonfun$publishDiagnostics$3$adapted(Diagnostics.scala:196)
	at scala.collection.Iterator.foreach(Iterator.scala:943)
	at scala.collection.Iterator.foreach$(Iterator.scala:943)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1431)
	at scala.collection.IterableLike.foreach(IterableLike.scala:74)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:73)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:56)
	at scala.meta.internal.metals.Diagnostics.publishDiagnostics(Diagnostics.scala:196)
	at scala.meta.internal.metals.Diagnostics.publishDiagnostics(Diagnostics.scala:174)
	at scala.meta.internal.metals.Diagnostics.didChange(Diagnostics.scala:122)
	at scala.meta.internal.metals.MetalsLanguageServer.didChange(MetalsLanguageServer.scala:900)
	... 15 more

2020.11.02 15:49:54 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
Nov 02, 2020 3:50:51 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 3:50:53 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 3:50:53 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

2020.11.02 15:51:45 WARN  no build tool detected in workspace '/home/seth/Desktop/Scala_Labs'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
2020.11.02 15:57:16 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 15:57:34 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 15:58:00 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 57)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 50)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 51)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 52)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 53)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[<error>]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 54)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[String]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 55)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[String]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 55)
something's wrong: no file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala in Array[String]RangePosition(file:///home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala, 42, 42, 55)
Nov 02, 2020 3:59:05 PM scala.meta.internal.pc.CompilerAccess handleError
SEVERE: offset 68, count -2, length 244
java.lang.StringIndexOutOfBoundsException: offset 68, count -2, length 244
	at java.base/java.lang.String.checkBoundsOffCount(String.java:3304)
	at java.base/java.lang.String.rangeCheck(String.java:280)
	at java.base/java.lang.String.<init>(String.java:276)
	at scala.tools.nsc.interactive.Global.typeCompletions$1(Global.scala:1230)
	at scala.tools.nsc.interactive.Global.completionsAt(Global.scala:1253)
	at scala.meta.internal.pc.SignatureHelpProvider.$anonfun$treeSymbol$1(SignatureHelpProvider.scala:376)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.pc.SignatureHelpProvider.treeSymbol(SignatureHelpProvider.scala:374)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCall$.unapply(SignatureHelpProvider.scala:199)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.visit(SignatureHelpProvider.scala:310)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.traverse(SignatureHelpProvider.scala:304)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.fromTree(SignatureHelpProvider.scala:273)
	at scala.meta.internal.pc.SignatureHelpProvider.signatureHelp(SignatureHelpProvider.scala:27)

2020.11.02 15:59:15 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:06:12 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
Nov 02, 2020 4:06:36 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:06:37 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:06:37 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:06:47 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:06:48 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:06:52 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:06:54 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:06:54 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:07:00 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

Nov 02, 2020 4:07:01 PM scala.meta.internal.pc.completions.Completions completionPosition
SEVERE: null
java.lang.NullPointerException
	at scala.meta.internal.pc.completions.OverrideCompletions$OverrideCompletion.<init>(OverrideCompletions.scala:46)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe(Completions.scala:478)
	at scala.meta.internal.pc.completions.Completions.completionPositionUnsafe$(Completions.scala:409)
	at scala.meta.internal.pc.MetalsGlobal.completionPositionUnsafe(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.completions.Completions.completionPosition(Completions.scala:401)
	at scala.meta.internal.pc.completions.Completions.completionPosition$(Completions.scala:386)
	at scala.meta.internal.pc.MetalsGlobal.completionPosition(MetalsGlobal.scala:29)
	at scala.meta.internal.pc.CompletionProvider.safeCompletionsAt(CompletionProvider.scala:444)
	at scala.meta.internal.pc.CompletionProvider.completions(CompletionProvider.scala:57)
	at scala.meta.internal.pc.ScalaPresentationCompiler.$anonfun$complete$1(ScalaPresentationCompiler.scala:184)
	at scala.meta.internal.pc.CompilerAccess.withSharedCompiler(CompilerAccess.scala:137)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$withInterruptableCompiler$1(CompilerAccess.scala:87)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:197)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:103)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

2020.11.02 16:12:04 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:12:34 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:17:14 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:18:30 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:18:42 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:26:44 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:27:12 WARN  no build tool detected in workspace '/home/seth/Desktop/Scala_Labs'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
2020.11.02 16:27:12 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:29:37 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:30:39 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:31:59 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:42:04 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:44:42 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
2020.11.02 16:45:17 WARN  no build target for: /home/seth/Desktop/Scala_Labs/Lab2/Temperature.scala
